### ISSUE TITLE
Does not accept null values

### ISSUE THREAD

**Issue by devfaysal** [calebporzio/sushi#38]
It looks like by default sushi makes all columns 'not null'.
Is there any way to define a schema for it so that sushi accepts some null values?
Or I am missing something?

```
protected $rows = [
	['id' => '1','name' => 'Comilla','bn_name' => 'কুমিল্লা','lat' => '23.4682747','lon' => '91.1788135'],
  	['id' => '2','name' => 'Feni','bn_name' => 'ফেনী','lat' => '23.023231','lon' => '91.3840844'],
  	['id' => '4','name' => 'Rangamati','bn_name' => 'রাঙ্গামাটি','lat' => NULL,'lon' => NULL]
]
```

This returns "NOT NULL constraint failed"

**Comment by faustbrian** [comment]
I ran into this before too and think the best solution would be to extract this https://github.com/calebporzio/sushi/blob/master/src/Sushi.php#L84-L100 into a `protected` function which can be overwritten to define a schema that should be used.

**Comment by devfaysal** [comment]
Great! But wouldn't it be nice if we can just use an array to define the nullable columns?

**Comment by faustbrian** [comment]
That would be one solution but what if you want more explicit control over your schema? Then you would have to maintain an implementation of the nullable array for some people and the schema method for others.

**Comment by ryangjchandler** [comment]
I agree with both of you on this one. There's definitely a new for granular control of the schema, but there should be shortcuts for specific more common things such as nullable columns, similar to how Laravel provides a `$dates` array for casting, but you could also just use the `$casts` property.

**Comment by calebporzio** [comment]
Fixed in 2.0.0

**Comment by calebporzio** [comment]
Also, you can now define a custom schema: https://github.com/calebporzio/sushi#custom-schema
